.evergreen/scripts/prepare-resources.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+. src/.evergreen/scripts/env.sh
+set -o xtrace
+rm -rf $DRIVERS_TOOLS
+if [ "$PROJECT" = "drivers-tools" ]; then
+    # If this was a patch build, doing a fresh clone would not actually test the patch
+    cp -R $PROJECT_DIRECTORY/ $DRIVERS_TOOLS
+else
+    git clone https://github.com/mongodb-labs/drivers-evergreen-tools.git $DRIVERS_TOOLS
+fi
+echo "{ \"releases\": { \"default\": \"$MONGODB_BINARIES\" }}" >$MONGO_ORCHESTRATION_HOME/orchestration.config
```

Comment:
We can make the call to https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/setup.sh in this file.